Create separate list of page-type values (#21272)
mdn/content: files/en-us/mdn/writing_guidelines/page_structures/page_types/index.md

## The page-type front matter key

> We have defined a front matter key `page-type` to describe the type of MDN pages. This project is a work in progress: so far we have only defined `page-type` values for the [CSS pages](/en-US/docs/Web/CSS) and the [Web API pages](/en-US/docs/Web/API). The templates linked below indicate which `page-type` values you should set for each page type.
> 
> For the complete list of page types see [The page-type front matter key](/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/Page_type_key).

Reviewer: FWIW I think we should only say "This project is a work in progress: so far we have only defined `page-type` values for the [CSS pages](/en-US/docs/Web/CSS) and the [Web API pages](/en-US/docs/Web/API)." once, and that should be in the doc itself. 

```suggestion
We have defined a front matter key `page-type` to clearly identify the type of MDN pages. The templates linked below indicate which `page-type` values you should set for each page type.

For the complete list of page types see [The page-type front matter key](/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/Page_type_key).
```

Author: Yeah, this was a copy/paste error.
